### ISSUE TITLE
F1 to open MSDN documentation only works in insert mode

### ISSUE THREAD

**Issue by BenjaminBrandtner** [VsVim/VsVim#2671]
An out of the box feature in Visual Studio is placing the cursor on any Class, Property, Keyword, etc. and pressing F1 to open the MSDN Documentation in your browser. I use this feature a lot.

In VsVim F1 doesn't work in Normal mode, you first have to enter Insert mode to get it to work.
I've used this extension for around 1.5 years now and don't think this was alway the case, I think it may have worked correctly in 2.6.0.0, but I can't say for sure.

I'm not loading any vimrcs.

**Environment (please complete the following information):**

- Visual Studio version:
Community 2019 (16.1.5)
and Enterprise 2017 (15.9.6)
- VsVim version:
2.7.0.0 and 2.8.245
- Programming Language: C#, probably others
- Check(Type 'x') any that are installed:
    - [ ] ReSharper
    - [ ] Visual Assist

**Comment by jaredpar** [comment]
Definitely did work in normal mode in the past. Know becuase i dislike this behavior and whenever MSDN opened it was a reminder that I needed to fix my vsvimrc file to disable it. :smile:

Will take a look.

**Comment by ricksladkey** [comment]
@jaredpar, I'm like you: I intentionally map `<F1>` to `<Nop>`! But it is true it **should** work if you want it to...

I introduced this bug with the Czech keyboard changes (as I recall) and I've already got a fix for it.

**Comment by ricksladkey** [comment]
I think this bug was introduced in PR #2516, which has a description of the related issues:

- https://github.com/VsVim/VsVim/pull/2516#issue-275528735

**Comment by BenjaminBrandtner** [comment]
I tried mapping <F1> to <Nop> before, it didn't help. With it, the cursor just moves one step forward when pressing F1 in Normal mode.

**Comment by ricksladkey** [comment]
@BenjaminBrandtner, this should be fixed in the next nightly build if you want to try it.

If you would rather use VsVim 2.7.0, the workaround is to provide an explicit mapping for the key like this in your rc file:

```
nnoremap <F1> :vsc Help.F1Help<CR>
inoremap <F1> :vsc Help.F1Help<CR>
```

and then close and re-open Visual Studio and mark `F1` as "Handled by VsVim" on the VsVim keyboard options page.

**Comment by akeeton** [comment]
@ricksladkey I'm also running into this problem.  I upgraded to version 2.8.271 but it still exhibits the same behavior.

**Comment by ricksladkey** [comment]
@akeeton It looks like the nightly builds are getting built for branches other than master and so the "latest" VSIX Gallery build #271 does not include the PR that is necessary for this to work. I've filed a separate issue #2685. After that is resolved, you can try to verify the verify the fix for the F1 help problem. Sorry about the confusion!

**Comment by ricksladkey** [comment]
@akeeton Jared fixed this and 2.8.276 is currently in the Open VSIX Gallery and corresponds to the head of master! I would be grateful if you could test the F1 problem.

**Comment by akeeton** [comment]
It works on version 2.8.278, thanks.
